test/e2e/tests/errors.spec.js

```diff
@@ -409,13 +409,19 @@ describe('Sentry errors', function () {
             (breadcrumb) =>
               breadcrumb.message.match(/(Running migration \d+)/u)[1],
           );
-          const firstMigrationLog = migrationLogMessages[0];
-          const lastMigrationLog =
-            migrationLogMessages[migrationLogMessages.length - 1];
+          // Temporarily allow either 8 or 10, until we get to the bottom of this
+          assert(
+            migrationLogMessages.length === 8 ||
```

Comment:
When running the test 30 consecutive times, these migrations were run:

26 times
migrationLogMessages [
'Running migration 75',
'Running migration 76',
'Running migration 77',
'Running migration 78',
'Running migration 79',
'Running migration 80',
'Running migration 81',
'Running migration 82'
]

2 times
migrationLogMessages [
'Running migration 75',
'Running migration 76',
'Running migration 77',
'Running migration 78',
'Running migration 79',
'Running migration 80',
'Running migration 81',
'Running migration 82',
'Running migration 83',
'Running migration 84'
]

2 times
migrationLogMessages [
'Running migration 75', 'Running migration 76',
'Running migration 77', 'Running migration 78',
'Running migration 79', 'Running migration 80',
'Running migration 81', 'Running migration 82',
'Running migration 83', 'Running migration 84',
'Running migration 85', 'Running migration 86',
'Running migration 87', 'Running migration 88',
'Running migration 89', 'Running migration 90',
'Running migration 91', 'Running migration 92',
'Running migration 92', 'Running migration 92',
'Running migration 92'
]

(the 4 migration 92 include 92, 92.1, 92.2 and 92.3)

So you might need:

```js
|| migrationLogMessages.length === 21,
```

Reply:
This interim fix also works for 30+ test runs https://github.com/MetaMask/metamask-extension/pull/21761/files

Reply:
Oh interesting, I never saw 21, but sure... I added it in.